model/otel/registry.yaml

```diff
@@ -71,6 +71,11 @@ groups:
         brief: The version of the instrumentation scope - (`InstrumentationScope.Version` in OTLP).
         examples: ['1.0.0']
         stability: stable
+      - id: otel.scope.schema_url
+        type: string
+        brief: The schema URL of the instrumentation scope.
```

Comment:
Schema URL is defined as part of instrumentation scope in the spec: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/common/instrumentation-scope.md

However, in OTLP it is always on a "higher" level for reasons unknown to me (maybe it was historically introduced before the concept of the instrumentation scope?):  https://github.com/open-telemetry/opentelemetry-proto/blob/c30610041736aa5c0077b156f27b09e878b797ea/opentelemetry/proto/trace/v1/trace.proto#L58-L64